policies/README.md

```diff
@@ -0,0 +1,107 @@
+
+## OPA POLICIES
+
+Policies are rules that can be applied when any of your cluster object is _created_, _updated_, _patched_ and _deleted_.
+
+Policies are always executed _before_ the action takes place and their goal is to allow or prevent the requested actions.
+
+The language used to express OPA policies is [REGO] (https://www.openpolicyagent.org/docs/latest/policy-language/).
```

Comment:
```suggestion
The language used to express OPA policies is [REGO](https://www.openpolicyagent.org/docs/latest/policy-language/).
```

Otherwise the link is not displayed correctly